database-mySql/index.js

```diff
@@ -106,7 +107,7 @@ User.sync({
 })
   .then(() => {
     // add the same user again to test function - should return false
-    User.findOrCreate({ where: { userName: 'Josef', email: 'email@email.com' } })
+    User.findOrCreate({ where: { username: 'Josef', email: 'email@email.com' } })
```

Comment:
hardcoded to one user

Reply:
I have never touched this file, talk to Joe about this. He did this. This just puts one user in the database every time we start the server/ database. To ensure we have atleast one test dummy.